pulp_rpm/tests/functional/api/test_consume_content.py

```diff
@@ -40,77 +50,103 @@ def setUpClass(cls):
         configuration = cls.cfg.get_bindings_config()
         core_client = CoreApiClient(configuration)
         cls.artifacts_api = ArtifactsApi(core_client)
+        cls.client = gen_rpm_client()
+        cls.repo_api = RepositoriesRpmApi(cls.client)
+        cls.remote_api = RemotesRpmApi(cls.client)
+        cls.publications = PublicationsRpmApi(cls.client)
+        cls.distributions = DistributionsRpmApi(cls.client)
         cls.before_consumption_artifact_count = 0
         cls.pkg_mgr = cli.PackageManager(cls.cfg)
         cls.pkg_mgr.raise_if_unsupported(unittest.SkipTest, "This test requires dnf or yum.")
 
     def _has_dnf(self):
         return self.pkg_mgr.name == "dnf"
 
-    def test_on_demand_policy(self):
-        """Verify whether content synced with on_demand policy can be consumed.
-
-        This test targets the following issue:
-
-        `Pulp #4496 <https://pulp.plan.io/issues/4496>`_
-        """
+    def test_on_demand_policy_mirror(self):
+        """Verify that content synced with on_demand policy in mirror mode can be consumed."""
         delete_orphans()
-        self.do_test("on_demand")
+        self.do_test("on_demand", True)
         new_artifact_count = self.artifacts_api.list().count
         self.assertGreater(new_artifact_count, self.before_consumption_artifact_count)
 
-    def test_streamed_policy(self):
-        """Verify whether content synced with streamed policy can be consumed.
+    def test_streamed_policy_mirror(self):
+        """Verify that content synced with streamed policy in mirror mode can be consumed."""
+        delete_orphans()
+        self.do_test("streamed", True)
```

Comment:
For readability, it would be helpful to have at least the mirror parameter named in all tests.
I can quickly guess that the first argument is a policy because of the value, but True/False one, I can guess only if I read the doc string or try to associate with the test name. But to be sure, I scroll to the `do_test` and verify my understanding anyway. :)